features/vm-service-credential-distribution.md

```diff
@@ -0,0 +1,225 @@
+[//]: # (The syntax preceeding this line is a comment marker used to help guide the author in populating this document)
+[//]: # (to github. Unlike HTML comments commonly used throughout istio.io documentation, this comment will not be rendered)
+[//]: # (by github. Comments must be separated by carriage return preceding and concluding the text and be a single line.)
+
+[//]: # (This is a living document representing the maturity of a feature. Completion of this template enables Istio work groups)
+[//]: # (to collect information on potential new functionality. This template should be completed before users are exposed to)
+[//]: # (any new experimental feature. Please complete this template during development.)
+
+[//]: # (The feature implementation section must be completed before submission of the document.)
+
+# Feature:
+
+[//]: # (All information in this section is mandatory.)
+
+**Feature name:**
+
+Service Credential Distribution for Virtual Machines
+
+[//]: # (The name of the feature, e.g. Multiple control planes)
+
+**Primary lead(s):**
+
+Limin Wang
+
+[//]: # (The primary lead or leads responsible for the feature. These individuals serve as a point of contact for the feature.)
+
+**Short description:**
+
+Identity and certificate management for VMs in Istio
+
+[//]: # (A short description of the feature. One or two sentences maximum.)
+
+
+**Design Docs:**
+
+- [Mesh Expansion: VM Security](https://docs.google.com/document/d/1-612Sgz_skeoX44dw3MU6Z8ONgq1f29wLUI9zggyLec/edit#heading=h.ls9r1x3pxz4v)
+- [VM Cert Provisioning](https://docs.google.com/document/d/1lPyIPLYQfjde_DUL3eBIlJqof65LSoDiRqB60xVZESY/edit#heading=h.xw1gqgyqs5b)
+- [VM Identity Bootstrapping](https://docs.google.com/document/d/1lPyIPLYQfjde_DUL3eBIlJqof65LSoDiRqB60xVZESY/edit#heading=h.xw1gqgyqs5b)
```

Comment:
This and the previous doc are the same. Did you mean for them to be different?

Reply:
Added a do-not-merge for this. Feel free to remove once you take a look. Other than that looks good. Thanks :)

Reply:
Fixed it. Was a copy-paste mistake. Thanks @brian-avery.